### PR TITLE
LPS-148319 Reorganize and remove unused CSS from Page Editor

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:relevant

**Comment by veroglez** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments-widgets/components/_TabItem.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@ html#{$cadmin-selector} {
 	.cadmin {
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments-widgets/components/_TabItem.scss:4]: Did you revise if all the CSS reamining here is necessary? I did that with the components I worked on and I found that some code wasn't necessary

**Review by sandrodw3 — COMMENTED** [review]
LGTM 🥰
Just left some suggestions!

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments-widgets/components/_TabItem.scss (hunk 1 above)
Comment (veroglez) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments-widgets/components/_TabItem.scss:4]: Yep, it seems that everything is necessary in this case

**Comment by veroglez** [comment]
ci:forward

**Pull request closed without merging**